docs/versioned_docs/version-v0.8/concepts/security.mdx

```diff
@@ -385,7 +385,7 @@ Allow the user to show the secret in the UI:
 :::warning
 
 This feature is a work in progress and is tracked as
-[kratos#133](https://github.com/ory/kratos/issues/138).
+[kratos#138](https://github.com/ory/kratos/issues/138).
```

Comment:
These changes are only made in the versioned docs?

Reply:
yes will do current and then mark ready for review ;)